### PR TITLE
Added conditional for ui slug validation

### PR DESCRIPTION
**Pull request by jonathanurias96**
#510

### CODE REVIEW

**Comment by brettski** [comment]
Hi @jonathanurias96. Your commit has linter errors. You can run `npm run lint` from the command line to verify. I am a bit surprised the hook didn't fire on push as well.  

Errors reported:
07:54:00.756  [lint] /vercel/3ed8b157/components/Member/Profile/ContactInfo.js
07:54:00.756  [lint]   119:19  error  'userSlug' is never reassigned. Use 'const' instead     prefer-const
07:54:00.756  [lint]   120:19  error  'isSlugValid' is never reassigned. Use 'const' instead  prefer-const
07:54:00.756  [lint]   120:67  error  Insert `;`                                              prettier/prettier
07:54:00.756  [lint]   126:17  error  Assignment to property of function parameter 'errors'   no-param-reassign

Thank you for all your time and efforts!

**Comment by jonathanurias96** [comment]
Got it. I will make the adjustments and let you know when I have it ready.

**Comment by jonathanurias96** [comment]
Ready for review

**Comment by brettski** [comment]
All checks passed!
I pulled the branch down and ran it locally. The check doesn't seem to fire. 
I login with a new account and go to enter my profile information. 
On the first page I enter my first name, last name and profile slug. For `Profile Slug` I enter `Brett$slaski` and click off the field. The check should happen at this point but does not. The slug I entered doesn't satisfy the RegExp and the field should turn red with error message and does not. 

I then tried to enter a slug I know already exists in the database and this did cause the field to show an error.

**Comment by jonathanurias96** [comment]
Got it. I will keep working on it!

**Comment by jonathanurias96** [comment]
The problem that I have is that since I can't access to that part of the website locally because of the request errors. I am not able to test it. So I am looking for a way that I can get to that page without any errors.

**Comment by jonathanurias96** [comment]
Ready for review

File: components/Member/Profile/ContactInfo.js [hunk 1]
```diff
@@ -70,8 +72,12 @@ const ContactInfoForm = ({
         return 'Required';
       }
 
+      if (isSlugInvalid) {
+        return 'Invalid characters';
```
Comment (brettski) [inline components/Member/Profile/ContactInfo.js:76]: Rename to: `Slug contains invalid characters`

File: components/Member/Profile/ContactInfo.js [hunk 2]
```diff
@@ -70,8 +72,12 @@ const ContactInfoForm = ({
...
+      }
+
       if (validProfileSlug === false) {
-        return 'Slug already taken or invalid characters';
+        return 'Slug already taken';
```
Comment (brettski) [inline components/Member/Profile/ContactInfo.js:80]: Please update to: `Slug is already taken`

**Comment by brettski** [comment]
Thank you @jonathanurias96! Please come join THAT Community in our slack channel: https://thatslack.thatconference.com

**Pull request merged**